Create Large Test Files.py

```diff
@@ -0,0 +1,255 @@
+# 
+# The following defines a function that creates clean test files. The function uses 1000 row clean test files located in the repository for Bank 0 and Bank 1 to duplicate LAR data to produce data with a desired number of rows, and provide unique ULI's for each row. Test files may be created with rows as multiples of 1000. 
+
+# In[299]:
+
+
+def large_clean_test_files(bank=None, file=file, count=None):
+    
+    """
+    Creates clean HMDA submission test files in multiples of 1000, specified by the count variable. 
+    The function requires a count, and a specification for the test bank, either '0' or '1.'
+    
+    The function replicates LAR data from two clean test files in the repository.
+    """
+   #Checks whether the count is a multiple of 1000.  
+    if (count % 1000) != 0:
+        statement = "Please specify a count in a multiple of 1000"
+        return print(statement)
+    #Establishes the case for bank = 0. 
+    if bank == 0:
+        file = "clean_file_1000_rows_Bank0.txt" 
+        with open(file, 'r+' ) as f: #Opens the 1000 row file for Bank 0. 
+            ts_row = f.readline() #Reading in the first row as the Transmittal Sheet submission. 
+            lar_rows = f.readlines() #Reading in the other rows as the LAR submission.  
+        with open("edits_files/lar_rows.txt", 'w') as out_lar: #Writes a file of LAR data. 
+            out_lar.writelines(lar_rows)
+        with open("edits_files/ts_row.txt", 'w') as out_ts: #Writes a file of TS data. 
+            out_ts.writelines(ts_row)
+        with open('schemas/lar_schema.json') as d: #Loads in the schema for the LAR file as a dataframe. 
+            data = json.load(d)
+            data = pd.DataFrame(data)
+
+        #Reads in the LAR data as a dataframe, as object values without converting NA values to 
+        #Not a Number Values. 
+        df = pd.read_csv('edits_files/lar_rows.txt', delimiter = "|", header = None, dtype = 'object',
+                         keep_default_na=False)
+        df.columns = data['field'] #Takes the field names from the LAR schema as column names. 
+        
+        #Concatenates data to produce the desired number of rows in a new LAR dataframe. 
+        df = pd.concat([df]*int((count/1000)), ignore_index = True) 
+        
+        #Creates a list of unique ULI's for Bank 0, and checks to determine that ULI's are unique.
+        uli_1 = {'uli':list(['B90YWS6AFX2LGWOXJ1LD']*int((count)))}
+        df1 = pd.DataFrame(uli_1)
+        df1["uli"] = df1.apply(lambda x: x.uli + char_string_gen(23), axis = 1)
+        df1['uli'] = df1.apply(lambda x: x.uli + check_digit_gen(ULI = x.uli), axis = 1)
+        uli_check(df1)
+
+        #Replaces ULI's in the new LAR dataframe with the unique ULI's.  
+        df['uli'] = df1['uli']
+        
+        #Creates a new LAR csv file with the new LAR data with unique ULI's. 
+        df.to_csv("lar_" + str(count) + ".txt", sep = "|", index = False, header= None)
```

Comment:
Per PEP 8, remove spaces around = operator in function call when using named arguments.